src/submit-general-duty-table.php

```diff
@@ -12,28 +14,36 @@
     $end = isset($_POST["end"]) ? $_POST["end"] : null;
     $location_id = isset($_POST["location_id"]) ? $_POST["location_id"] : null;
 
+    // Check if empID is empty
+    if (empty($empID)) {
+        $response = array("status" => "error", "message" => "Employee ID is required");
+        echo json_encode($response);
+        exit; // Terminate the script
+    }
+
     // Create a new DbConnector instance
-    $dbConnector = new DbConnector();
+    $dbConnector = new DBConnector();
 
     try {
+        
         // Check if the empID already exists in the database
         $checkSql = "SELECT COUNT(*) as count FROM duty WHERE empID = :empID";
-        $checkStmt = $dbConnector->getConnection()->prepare($checkSql);
+        $checkStmt = $dbConnector->con->prepare($checkSql);
```

Comment:
The DBConnector class does not have a property named con
It returns the connection (con) through the getConnection method
So this won't work at all :/